pkg/systemd/overview-cards/healthCard.scss

```
@@ -24,6 +24,10 @@
       > :not(a):last-child {
         flex: auto;
       }
+
+      .pf-v6-l-flex {
+        gap: var(--pf-v6-global--spacer--xs) var(--pf-v6-global--spacer--sm);
+      }
```

Looks like a good approach for now! There is a difference between PF5 and PF6 so the naming did change a bit:
```suggestion

      .pf-v6-l-flex {
        gap: var(--pf-t--global--spacer--xs) var(--pf-t--global--spacer--sm);
      }
```